test_model.py

```diff
@@ -41,6 +26,7 @@ def get_vectorial_results(queries, inv_index, stop_words, stats_collection, args
     if args.model == "boolean":
         inv_index = pickle_load_from_file(config.get('simple_index_path', 'simple_index_path'))
         for query in queries_and_output:
+            query_stop_words = remove_query_from_stop_words(query, stop_words)
```

Comment:
This is not `query_stop_words`, it is `stop_words_not_in_query`, right ?
And it doesn't seem to be used anywhere.

Reply:
Yes the name isn’t great, I’ll change it, and I forgot to change the function call for the boolean model.

Reply:
Thanks 👍